### ISSUE TITLE
Will PandaCSS implementation be included in docs?

### ISSUE THREAD

**Issue by ilterugur** [mantinedev/mantine-v7#7]
Hello, thank you so much for your great work.

I see CSS Modules and Vanilla Extract are the recommended & described options included in the docs. Will you also add PandaCSS? It seems very promising.

**Comment by rtivital** [comment]
No, it will not be included in the docs. You can use it in your projects like any other library – there are no limitations from Mantine side.

**Comment by xhttp-res** [comment]
Mantine v7 use pandaCSS?
